include/Flow.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "Packet.hpp"
+#include "Server.hpp"
+
+#include <cstdint>
+#include <vector>
+#include <queue>
+
+/**
+ * Packet flow between source and destination nodes (servers).
+ */
+class Flow {
+public:
+    explicit Flow(const std::vector<std::uint32_t>& distances);
+    Flow() = default;
+    ~Flow() = default;
+
+   /**
+    * @brief Send packets from every server sender in the amount of CWND size.
+    */
+    void Send();
+
+    /**
+    * @brief Last ID given to network device.
```

Comment:
Which network device? Should this be in a Flow class?

Reply:
No, I will move it to NetworkSimulator class.